Makefile

```diff
@@ -76,6 +76,26 @@ package-dashboard-rpm:
 
 	rm -rf ${PWD}/build
 
+### build rpm for apisix-openresty:
+.PHONY: package-apisix-openresty-centos7-rpm
```

Comment:
Better to use package-apisix-openresty-rpm to keep the name consistent

Reply:
update